src/cloudflare/internal/test/images/images-api-test.js

```diff
@@ -135,9 +135,7 @@ export const test_images_transform_consumed = {
     let e;
 
     try {
-      let transformer = env.images
-        .input(blob.stream())
-        .transform({ rotate: 90 });
+      let transformer = env.images.input(blob.stream()).transform({ rotate: 90 });
```

Comment:
For me, the way it was before formatting is clearer.

Reply:
Agreed. Maybe we should reduce the maximum line length a bit?

Reply:
Line length adjustment fixed this one